### PR TITLE
Authority claimer with dispatcher for release 1.2

### PR DESCRIPTION
**Pull request by GMKrieger**
#130

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,11 +11,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added `cartesi-rollups-node` Go binary as a single entrypoint to execute all Cartesi Node services
 - Added `authority-claimer` service
```
Comment (marcelstanley) [inline CHANGELOG.md:13]: Perhaps we should make it clear that the dispatcher no longer sends claims

**Review by marcelstanley — COMMENTED** [review]
Looks good! Other than the comment below, I believe we could add a `!` to the commit message as well

File: offchain/authority-claimer/src/checker.rs [hunk 2]
```diff
@@ -88,7 +88,7 @@ impl DuplicateChecker for DefaultDuplicateChecker {
         rollups_claim: &RollupsClaim,
     ) -> Result<bool, Self::Error> {
         let proof_context =
-            U256([0, 0, 0, rollups_claim.epoch_index]).encode().into();
```
Comment (gligneul) [inline offchain/authority-claimer/src/checker.rs:91]: Is this change intended? If yes, please add it in another commit.

**Comment by gligneul** [comment]
Also, let's test this on rollups-examples before merging. I want to avoid breaking the main branch by accident.

File: offchain/authority-claimer/src/checker.rs (hunk 2 above)
Comment (GMKrieger) [reply offchain/authority-claimer/src/checker.rs:91]: It's intentional. The way it was before, it was building the proof hex the other way around, e.g. 0x01000000 instead of 0x00000001. The claim contracts were not happy with this, as both the get couldn't find it and submit thought we skipped epochs.

**Review by marcelstanley — APPROVED** [review]
